### PR TITLE
feat: linea bridge balance mismatch moved to l2-bridge-linea

### PR DESCRIPTION
**Pull request by sergeyWh1te**
Part of https://github.com/lidofinance/alerting-forta/issues/526

### CODE REVIEW

File: l2-bridge-linea/src/clients/eth_provider_client.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+export class ETHProvider implements IL1BridgeBalanceClient {
+  private readonly wStEthRunner: wStEthRunner
+  private readonly logger: Logger
+
+  constructor(logger: Logger, wStEthRunner: wStEthRunner) {
+    this.wStEthRunner = wStEthRunner
+    this.logger = logger
+  }
+
+  public async getWstEth(l1blockNumber: number, address: string): Promise<E.Either<Error, BigNumber>> {
```
Comment (TheDZhon) [inline l2-bridge-linea/src/clients/eth_provider_client.ts:21]:
```suggestion
  public async getWstEthBalance(l1blockNumber: number, address: string): Promise<E.Either<Error, BigNumber>> {
```

File: l2-bridge-linea/src/services/bridge_balance.ts [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+          wStEth_L1.left,
+          `Error in ${BridgeBalanceSrv.name}.${this.handleBridgeBalanceWstETH.name}:36`,
+          `Could not call clientL1.getWstEth`,
+          l1BlockNumber,
+        ),
+      ]
+    }
+
+    for (const l2blockNumber of l2BlockNumbers) {
+      const wStEth_L2 = await this.clientL2.getWstEth(l2blockNumber)
```
Comment (TheDZhon) [inline l2-bridge-linea/src/services/bridge_balance.ts:62]:
```suggestion
      const wstETHTotalSupply_onLinea = await this.clientL2.getWstEth(l2blockNumber)
```

**Review by TheDZhon — APPROVED** [review]
:+1: I'd suggest made a few name edits

File: l2-bridge-linea/src/services/bridge_balance.ts [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    const start = new Date().getTime()
+
+    const findings = await this.handleBridgeBalanceWstETH(l1BlockNumber, l2BlockNumbers)
+
+    this.logger.info(elapsedTime(this.name + '.' + this.handleBlock.name, start))
+    return findings
+  }
+
+  private async handleBridgeBalanceWstETH(l1BlockNumber: number, l2BlockNumbers: number[]): Promise<Finding[]> {
+    const wStEth_L1 = await this.clientL1.getWstEth(l1BlockNumber, this.lineaL1TokenBridge)
```
[the remark this answers is not in the document]
Comment (sergeyWh1te) [reply l2-bridge-linea/src/services/bridge_balance.ts]: b7ea207f618f65aa7607140a0be330a22a21e076

**Review by TheDZhon — APPROVED** [review]
:+1:

**Pull request merged**